src/equity/equity.controller.ts

```diff
@@ -0,0 +1,22 @@
+import { Controller, Get, UseGuards, Req } from '@nestjs/common';
+import { Request } from 'express';
+import { Local } from '~/decorator';
+import { JwtMember } from '~/auth/auth.dto';
+import { ProgramService } from '~/program/program.service';
+import { AuthGuard } from '~/auth/auth.guard';
+
+@UseGuards(AuthGuard)
+@Controller({
+  path: 'equity',
+  version: '2',
+})
+export class EquityController {
+  constructor(private programService: ProgramService) {}
+
+  @Get('/programs')
+  async getProgramsByMemberId(@Local('member') member: JwtMember, @Req() request: Request) {
```

Comment:
please add response type to handler function , this will help us to understand the return format